recipes/trace-remote-sampling/collector-config.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+spec:
+  image: otel/opentelemetry-collector-contrib:latest
+  # We need to specify ports so remote sampler is exposed.
+  ports:
+    - port: 4317
+      name: otlp
+    - port: 4318
+      name: otlp-grpc
+    - port: 5778
+      name: jaeger
+    - port: 14250
+      name: jaeger-grpc
+  # Connect the config-map w/ our jaeger sampler config so we can update it quickly.
+  volumes:
+  - name: sampling-config-volume
+    configMap:
+      name: remote-sampling-config
+  volumeMounts:
+  - name: sampling-config-volume
+    mountPath: /etc/otel/sampling
+    readOnly: true
+  # Ensure the jaeger remote sampler config is enabled as an extension.
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    exporters:
+      debug:
+    extensions:
+      jaegerremotesampling:
+        source:
+          reload_interval: 1s
```

Comment:
nit could probably go longer than 1s here. Was there a reason for such a short interval otherwise?